querydsl-tooling/querydsl-ksp-codegen/src/test/kotlin/RenderTest.kt

```diff
@@ -0,0 +1,202 @@
+import com.querydsl.core.types.dsl.BooleanPath
+import com.querydsl.ksp.codegen.QProperty
+import com.querydsl.ksp.codegen.QPropertyType
+import com.querydsl.ksp.codegen.QueryModel
+import com.querydsl.ksp.codegen.QueryModelRenderer
+import com.querydsl.ksp.codegen.SimpleType
+import com.squareup.kotlinpoet.ClassName
+import com.squareup.kotlinpoet.asClassName
+import org.junit.Assert
```

Comment:
Can we use assertJ assertions instead?

Reply:
Unclear which package you are referring to here.
I see that another module is using org.assertj.core.api.Assertions
is that the one you want?

Reply:
Oh actually nvm, it's obvious i noticed you wrote assertj now, it wasn't a typo :P